### ISSUE TITLE
Click tracking

### ISSUE THREAD

**Issue by dolphinsd** [tedd/Tedd.ShortUrl#19]
Did you consider adding built in tracking for clicks?

**Comment by tedd** [comment]
Yes, the previous version has this and I was planning on adding the ability for link creator to track clicks anonymously as well. But I have very little time to work on this at the moment.
